.env.example

```
@@ -0,0 +1,40 @@
+# python -c "import secrets; print(secrets.token_hex(16))"
+FLASK_SECRET_KEY=
+
+MAIN_DIR=~/data
+
+DB_NAME=svg_langs
+
+# DB_HOST=tools.db.svc.wikimedia.cloud
+DB_HOST=127.0.0.1
+
+TOOL_REPLICA_USER=root
+TOOL_REPLICA_PASSWORD=root
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The TOOL_REPLICA_PASSWORD key should go before the TOOL_REPLICA_USER key

(UnorderedKey)